runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -39,13 +38,14 @@ export function get_page_handler(
 	function handle_error(req: Req, res: Res, statusCode: number, error: Error | string) {
 		handle_page({
 			pattern: null,
+			resources: [],
```

Comment:
This is only used by my rollup plugin, probably shouldn't be here

Reply:
done